medical_prescription_sale_stock_us/models/medical_prescription_order_line.py

```diff
@@ -31,27 +30,31 @@ class MedicalPrescriptionOrderLine(models.Model):
         compute='_compute_can_dispense_and_qty',
         help='Total units allowed in the prescription, including refills',
     )
-    last_dispense_remain_qty = fields.Float(
+    dispense_remain_qty = fields.Float(
         string='Dispense Remaining Qty',
         compute='_compute_dispense_remain',
-        help='Estimated number of units remaining from last dispense',
-    )
-    last_dispense_remain_percent = fields.Float(
-        string='Dispense Remaining Percent',
-        compute='_compute_dispense_remain',
-        help='Estimated percentage remaining from last dispense',
+        help='Estimated number of units remaining from all finished dispenses',
     )
-    last_dispense_remain_day = fields.Float(
+    dispense_remain_day = fields.Float(
         string='Dispense Remaining Days',
         compute='_compute_dispense_remain',
-        help='Estimated days remaining from last dispense',
+        help='Estimated days remaining based on all finished dispenses',
     )
 
     @api.multi
+    @api.depends(
```

Comment:
This is going to create a conflict w/ https://github.com/laslabs/vertical-medical/blob/release/9.0/medical_prescription_sale_stock_us/models/medical_prescription_order_line.py#L51

`api.depends` needs to be redefined in all child methods, so it's not very appealing.


Was there a functional reason for these? Depends are only truly needed for store=True. The other advantage is the auto onchange, but we could easily add an onchange method to simply update attrs when dependents are changed.